modules/unsupported/mongodb/src/test/java/org/geotools/data/mongodb/FilterToMongoTest.java

```diff
@@ -54,4 +55,13 @@ public void testBBOX() throws Exception {
         assertNotNull(obj);
         System.out.println(obj);
     }
+
+    public void testLike() throws Exception {
+        PropertyIsLike like = ff.like(ff.property("stringProperty"), "on%", "%", "_", "\\");
+        BasicDBObject obj = (BasicDBObject) like.accept(filterToMongo, null);
+
+        assertNotNull(obj);
+        System.out.println(obj);
```

Comment:
Please do not include system outs in test cases.

You may wish to add a second test here to try out an expression that is not a propertyName.